src/reqstool/resources/schemas/v1/requirements.schema.json

```diff
@@ -257,8 +257,7 @@
                         "requirement_ids": {
                             "type": "array",
                             "items": {
-                                "$ref": "common.schema.json#/$defs/reqid",
-                                "description": "Requirement id that links to another requirement in this document"
+                                "$ref": "common.schema.json#/$defs/urnid"
```

Comment:
Is this the only place where urnid is used? I thought we used it for in SVCs as well. Have you checked the source code for the use of UrnId datatype?

Reply:
I think that you may be right. I'll take another look.